lib/Doctrine/Migrations/Tools/Console/Command/DiffCommand.php

```diff
@@ -22,6 +22,9 @@
  */
 class DiffCommand extends AbstractCommand
 {
+    /** @var string */
```

Comment:
Is the comment really necessary?

Reply:
Yes: while the initial type could be inferred from the initial assignment, this isn't reliable enough to satisfy PHPStan. Thus, the type hint still needs to be included.